README.md

```diff
@@ -168,6 +168,9 @@ onEditChange(initialData, handleEditChange);
 You will add data attributes to your rendered block html so hydra knows where they are on the page and it
 will automatically handle click events and show a quanta toolbar when selecting a block.
 
+Add the `data-block-uid={}` attribute to your rendered block html and call the `enableBlockClickListener()` provided by 'hydra.js' whenever your component is mounted.
```

Comment:
doesn't it need to include the block id? 
Also they need to know its goes on the outer most container of the block

Reply:
An example might help also